awx/main/wsrelay.py

```diff
@@ -107,6 +107,9 @@ def cancel(self):
         self.async_task.cancel()
 
     async def run_connection(self, websocket: aiohttp.ClientWebSocketResponse):
+        producer = self.event_loop.create_task(self.run_producer("metrics", websocket, "metrics"))
+        self.producers["metrics"] = {"task": producer, "subscriptions": {"metrics"}}
+        logger.info("Producer metrics started.")
```

Comment:
Not sure how useful this log message is. This one here should do be enough, I think? 

https://github.com/ansible/awx/blob/43db246493ae4e1e4076341d44c6c3d989636d89/awx/main/wsrelay.py#L156